docs/uab_cloud/project_space_mgmt.md

```diff
@@ -0,0 +1,106 @@
+1. Your Lab can scale their project resources down or up dependent on research needs with ease and without significant delays, this way you avoid incurring additional costs for purchasing new hardware or under utilize already purchased resources.
+1. Created `Instances` in your project spaces provide security and privacy that can help to further protect your research data.
+
+The benefits of creating and using a project space for your Lab are way more than those mentioned, they are all geared towards improving your collaboration in producing high quality research.
+
+## How Do I Create a Project Space for My Research Lab
+
+To use the resources available on UAB Cloud.rc, you will need to send in a request to the UAB IT Research Computing Team via email [support@listserv.uab.edu](mailto:support@listserv.uab.edu). In the email, please state clearly your needs and the resources you would require for your lab. Your request should also include members of your lab to be included in the project space, and a preferred name (usually same as lab project folder on Cheaha). Please follow the naming conventions for requesting a [shared storage](../data_management/storage.md#how-do-i-request-shared-storage) on our platforms.
+
+## How Do I Switch Project Spaces?
+
+As a UAB Cloud.rc user, you can easily switch between your personal project space and other Lab project spaces you are a part of. From the dashboard of the homepage after login, navigate to the `Domain` and `Projects` drop down button, located in the top pane (upper left quadrant of the page) as shown in the image below. You can then select from the list of project spaces you belong to. The project space in your `blazerID` is your personal project space, other project spaces listed are the project spaces for Labs or Projects you are a part of. As of this time only a Lab Principal Investigator (PI), and/or a Research Core Director can request for a shared project space on Cloud.rc.
+
+![!RC Dashboard Screenshot showing Project Spaces](images/rc_move_project.png)
+
+## Creating an Instance in a Shared Project Space
+
+The steps for creating an `Instance` in a project are the same as creating an `Instance` in your personal project space, you can find a detailed guide here in our [Cloud Tutorial](tutorial/index.md). You would however, need to contact us to create a project space for your lab, as well as users who would be members of the shared project space.
+
+All members of a project space can see (and manage) all `Instances` within the shared project space on UAB Cloud.rc. They can do this by navigating through the landing page that doubles as the dashboard, the image below highlights the exact section. While members can see and create images or snapshots of an `Instance`, not all members can access the `Instance`. To do this, `SSH` key pairs would have to be created and added, see section on [Using a Key Pair to SSH](#using-a-key-pair-to-ssh).
```

Comment:
`ssh` in lower case, like the command is typed in Linux. Or without backticks in upper case, like the initialism "**S**ecure **SH**ell". I prefer SSH over `ssh`, except in code blocks.